### ISSUE TITLE
com.docker.hyperkit 100% cpu usage mac big Sur 11.4

### ISSUE THREAD

**Issue by ntedgi** [docker/for-mac#5796]
- [X] I have tried with the latest version of Docker Desktop
  - [X] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics

Expected behavior
normal cpu usgae
### Actual behavior
<img width="550" alt="Screen Shot 2021-06-29 at 9 43 53" src="https://user-images.githubusercontent.com/31243793/123750116-e27a8c00-d8be-11eb-9b7a-33c18c023cb3.png">

### Information

  - macOS Version:  big Sur 11.4 
  - Intel chip or Apple chip: Intel
  - Docker Desktop Version: 
  - 
<img width="584" alt="Screen Shot 2021-06-29 at 9 45 29" src="https://user-images.githubusercontent.com/31243793/123750042-c8d94480-d8be-11eb-928c-f441932e772c.png">

**Comment by rekham1077** [comment]
I am having a kind of similar issue, but with memory. It says com.docker.hyperkit is using 12.57GB/16GB of my memory even when I don't have any containers open or running

**Comment by enekochan** [comment]
What has worked for me (kind of) is increasing the resources available for Docker. In a 2,6GHz Dual Core i5 with 8GB RAM:

- 4 CPU
- 4GB RAM
- 2GB Swap
- 24GB disk image size

I get now around 20% CPU usage when my 8 container Docker is idle after you let it boot and stabilize.

**Comment by ntedgi** [comment]
![Screen Shot 2021-06-30 at 10 28 34](https://user-images.githubusercontent.com/31243793/123919768-f3460300-d98d-11eb-84a4-dfb134c97182.png)

still 100% cpu

**Comment by stephen-turner** [comment]
@rekham1077 Yes, it's normal. On Mac, Docker runs in a VM (because Mac can't run Linux containers natively). There is no way to scale up and down the memory taken by the VM, so we have to reserve it all up front.
